src/store/modules/rate/index.js

```diff
@@ -41,22 +41,26 @@ const actions = {
         })
     })
   },
-  getHistoryRates ({ commit }, { date, coin }) {
+  getHistoryRates ({ state, commit }, { date, coin }) {
     const url = getEnpointUrl('infoservice')
     const key = date
```

Comment:
What is it for? `const key = date`
Use `date`. Even better rename `date` param to `timestamp`